dz_1/dz_1_3.py

```diff
@@ -0,0 +1,16 @@
+"""
+Задача 6: Вы пользуетесь общественным транспортом? 
+Вероятно, вы расплачивались за проезд и получали билет с номером. 
+Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+*Пример:*
+385916 -> yes
+123456 -> no
+"""
+
+number = input("Введите номер вашего билета: ")
+
+if (len(number) == 6) & (int(number[0]) + int(number[1]) + int(number[2]) == int(number[3]) + int(number[4]) + int(number[5])):
```

Comment:
Побитовые операторы использовать можно, не рекомендуются. Они медленнее, чем обычные логические, так как для их работы операнды конвертируются в двоичное представление (если они изначально ен были таковыми) и уже потом происходит возврат выражения. Да, и скобки не нужны.
```suggestion
if len(number) == 6 and int(number[0]) + int(number[1]) + int(number[2]) == int(number[3]) + int(number[4]) + int(number[5]):
```